src/main/java/frc/robot/RobotContainer.java

```diff
@@ -38,19 +44,24 @@
  */
 public class RobotContainer {
   // The robot's subsystems
-  private final DriveSubsystem m_robotDrive = new DriveSubsystem();
+  public final DriveSubsystem m_robotDrive = new DriveSubsystem();
   private final EndEffectorIntake m_intake = new EndEffectorIntake();;
   private final LiftArm m_arm = new LiftArm();
+  private final Vision m_Vision = new Vision();
+  private final PhotonCamera m_camera = new PhotonCamera(VisionConstants.SnakeEyesCamera);
```

Comment:
Remove redundant `m_camera`

Reply:
You need to declare this i'm pretty sure, same line was in our code last year too

Reply:
the camera is declared in `Vision.java` - doesn't need to be declared again in RobotContainer

Reply:
Both are used for the TapeAlign command